.github/workflows/platform-docker-publish-all-features-image.yml

```diff
@@ -108,3 +121,16 @@ jobs:
           build-args: |
             SAML_INSTALLED=1
             POETRY_OPTS=--with saml,auth-controller
+
+  stop-runner:
+    needs: [start-runner, build-dockerhub]
+    if: ${{ always() }}
```

Comment:
Just curious, why is the `if:` here even necessary?

Reply:
Without it the `stop-runner` job won't get launched in case of failures in any of the prior jobs.

Reply:
I had pretty much the same question on the similar PR in the edge API repo 😄